src/components/cart.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
general comment: I don't know how you've managed to achieve so much of the criteria and extension criteria in such a succinct and clean way! Its amazing!

Reply:
Thank u so much, Marian!